node/chainSimulator/chainSimulator.go

```diff
@@ -36,6 +37,7 @@ type simulator struct {
 	initialWalletKeys      *dtos.InitialWalletKeys
 	nodes                  map[uint32]process.NodeHandler
 	numOfShards            uint32
+	mutex                  sync.RWMutex
```

Comment:
missing mutex lock on `GetRestAPIInterfaces` and `Close` methods

Reply:
done